src/components/ScholarshipAmountField.jsx

```diff
@@ -11,40 +11,48 @@ import RemoveIcon from '@material-ui/icons/Remove';
 import PropTypes from 'prop-types';
 import AmountType from '../types/AmountType';
 import AmountTextField from './AmountTextField';
+import { getIn } from 'formik';
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles(() => ({
   amountSelect: {
     minWidth: 150,
   },
   dash: {
-    marginTop: theme.spacing(2),
-  },
-  inputLabel: {
-    marginBottom: theme.spacing(1),
+    height: '100%',
+    textAlign: 'center',
   },
 }));
 
 function ScholarshipAmountField(props) {
   const classes = useStyles();
-  const {
-    helperText,
-    amountType,
-    minAmount,
-    maxAmount,
-    onTypeChange,
-    updateAmount,
-    labelStyle,
-  } = props;
+  const { labelStyle, formik } = props;
+
+  function updateAmount(min, max) {
+    formik.setFieldValue('amount.min', min, true);
+    formik.setFieldValue('amount.max', max, true);
+  }
+
+  function onTypeChange(e) {
+    formik.setFieldValue('amount.type', e.target.value, true);
+  }
+
+  const amountType = formik.values.amount.type;
+  const minAmount = formik.values.amount.min;
+  const maxAmount = formik.values.amount.max;
```

Comment:
With Formik doing the validation + suggested changes these are only used in one place so they can be inlined as well